### ISSUE TITLE
Some question with training and validation

### ISSUE THREAD

**Issue by Neopolitan-1999** [bowang-lab/U-Mamba#72]
hello,
I found u-mamba point fold all in instruction to train the model. May I ask if the validation will be all training data or random chosen from training data by program if we follow this instruction.

**Comment by JunMa11** [comment]
In nnU-Net's framework, the validation set is all the training cases in fold all.

**Comment by Neopolitan-1999** [comment]
> In nnU-Net's framework, the validation set is all the training cases in fold all.

Thank u for your reply, it helps a lot. But won't this cause the program to overfit if we set all training cases as validation?
